core/server/data/migrations/versions/4.17/02-add-offers-table.js

```diff
@@ -0,0 +1,18 @@
+const utils = require('../../utils');
+
+module.exports = utils.addTable('offers', {
```

Comment:
Oh I just realised - this is missing a column for `stripe_coupon_id`!

Reply:
Done